Library/Homebrew/extend/os/linux/keg_relocate.rb

```diff
@@ -41,8 +41,9 @@ def change_rpath(file, old_prefix, new_prefix)
     new_rpath = rpath.join(":")
     cmd = [patchelf, "--force-rpath", "--set-rpath", new_rpath]
 
-    if file.with_interpreter?
-      old_interpreter = Utils.safe_popen_read(patchelf, "--print-interpreter", file).strip
+    old_interpreter = file.interpreter
+    new_interpreter = old_interpreter
```

Comment:
```suggestion
```
Please remove this line.

Reply:
it is for safety purpose, 

consider the case when  `old_intepreter.present? ` is `false`
as `new_interpreter` is conditionally defined the line #55 will fail with undefined error

Reply:
Hmm. Good catch. In that case, I suggest…
```suggestion
    new_interpreter = nil
```

Reply:
i went with explicit check in the conditional below,
that explains the intention better than holding proxy values.